### ISSUE TITLE
Typo in method name

### ISSUE THREAD

**Issue by rickpastoor** [seivan-archives/SHSegueBlocks#2]
```
-(void)SH_performSegueWithIdentifier:(NSString *)theIdentifier
       andDestionationViewController:(SHPrepareForSegueDestinationViewController)theBlock;
```

should be

```
-(void)SH_performSegueWithIdentifier:(NSString *)theIdentifier
       andDestinationViewController:(SHPrepareForSegueDestinationViewController)theBlock;
```

**Comment by seivan** [comment]
Embarrassing. Damn you.

**Comment by seivan** [comment]
Fixed in 1.1.1. Thanks for bringing it up. Now, don't tell anyone :) 
:+1:

**Comment by rickpastoor** [comment]
No worries, great pod!
